src/briefcase/platforms/macOS/__init__.py

```diff
@@ -37,12 +40,96 @@ class macOSMixin:
     supported_host_os_reason = "macOS applications can only be built on macOS."
 
 
+class macOSInstallMixin(AppPackagesMergeMixin):
+    def _install_app_requirements(
+        self,
+        app: AppConfig,
+        requires: list[str],
+        app_packages_path: Path,
+    ):
+        # Perform the initial install targeting the current platform
+        host_app_packages_path = (
+            self.bundle_path(app) / f"app_packages.{self.tools.host_arch}"
+        )
+        super()._install_app_requirements(
+            app,
+            requires=requires,
+            app_packages_path=host_app_packages_path,
+        )
+
+        # Find all the packages with binary components.
+        # We can ignore any -universal2 packages; they're already fat.
+        binary_packages = self.find_binary_packages(
+            host_app_packages_path,
+            universal_suffix="_universal2",
+        )
+
+        # Now install dependencies for the architecture that isn't the host architecture.
+        other_arch = {
+            "arm64": "x86_64",
+            "x86_64": "arm64",
+        }[self.tools.host_arch]
+
+        # Create a temporary folder to hold wheels for other targets
+        other_app_packages_path = self.bundle_path(app) / f"app_packages.{other_arch}"
```

Comment:
What stops these temporary folders from being included in the final app?

Reply:
Location. The final directory structure for an app is something like:

- build/helloworld/macos/app
   - app_packages.arm64
   - app_packages.x86_64
   - support
   - Hello World.app
     - Contents
         - MacOS
         - Resources
            - app
            - app_packages
            - support